ml-agents/mlagents/trainers/tests/torch/test_ppo.py

```diff
@@ -0,0 +1,208 @@
+import pytest
+
+import numpy as np
+from mlagents.tf_utils import tf
+import copy
+import attr
+
+from mlagents.trainers.ppo.optimizer_torch import TorchPPOOptimizer
+from mlagents.trainers.policy.torch_policy import TorchPolicy
+from mlagents.trainers.tests import mock_brain as mb
+from mlagents.trainers.tests.test_trajectory import make_fake_trajectory
+from mlagents.trainers.settings import NetworkSettings
+from mlagents.trainers.tests.test_simple_rl import PPO_CONFIG
+from mlagents.trainers.tests.test_reward_signals import (  # noqa: F401; pylint: disable=unused-variable
+    curiosity_dummy_config,
+    gail_dummy_config,
+)
+
+
+@pytest.fixture
+def dummy_config():
+    return copy.deepcopy(PPO_CONFIG)
+
+
+VECTOR_ACTION_SPACE = 2
+VECTOR_OBS_SPACE = 8
+DISCRETE_ACTION_SPACE = [3, 3, 3, 2]
+BUFFER_INIT_SAMPLES = 64
+NUM_AGENTS = 12
+
+
+def create_ppo_optimizer_mock(dummy_config, use_rnn, use_discrete, use_visual):
```

Comment:
This is not really a mock optimizer. Do you think this can be renamed to create_test_ppo_optimizer ?